cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeJobCleaner.java

```diff
@@ -88,6 +92,19 @@ public long run(MasterEnvironmentContext context) {
                 LOG.warn("Failed to cleanup job resources for job {}. This attempt will be retried later.", jobName, e);
               }
             }
+            try {
+              coreV1Api = getCoreV1Api();
+              String configMapName = KubeTwillPreparer.CONFIGMAP_NAME_PREFIX +
+                job.getMetadata().getLabels().get(KubeTwillRunnerService.RUN_ID_LABEL);
+              coreV1Api.deleteNamespacedConfigMap(configMapName, kubeNamespace, null, null,
+                                                  null, null, null, null);
+            } catch (ApiException e) {
+              if (e.getCode() == 404) {
+                LOG.trace("Ignoring configmap deletion for job {} because configmap was not found", jobName, e);
+              } else {
+                LOG.warn("Failed to cleanup configmap for job {}.", jobName, e);
```

Comment:
log message should include e.getResponseBody(), otherwise we won't know what actually failed.

Reply:
fixed